index.html

```diff
@@ -23,7 +23,7 @@
eo.eu-central-1.amazonaws.com https://maps.geo.eu-west-1.amazonaws.com https://maps.geo.eu-west-2.amazonaws.com https://maps.geo.eu-north-1.amazonaws.com https://maps.geo.sa-east-1.amazonaws.com https://places.geo.us-east-1.amazonaws.com https://places.geo.us-east-2.amazonaws.com https://places.geo.us-west-2.amazonaws.com https://places.geo.ap-southeast-1.amazonaws.com https://places.geo.ap-southeast-2.amazonaws.com https://places.geo.ap-northeast-1.amazonaws.com https://places.geo.ca-central-1.amazonaws.com https://places.geo.eu-central-1.amazonaws.com https://places.geo.eu-west-1.amazonaws.com https://places.geo.eu-west-2.amazonaws.com https://places.geo.eu-north-1.amazonaws.com https://places.geo.sa-east-1.amazonaws.com https://routes.geo.us-east-1.amazonaws.com https://routes.geo.us-east-2.amazonaws.com https://routes.geo.us-west-2.amazonaws.com https://routes.geo.ap-southeast-1.amazonaws.com https://routes.geo.ap-southeast-2.amazonaws.com https://routes.geo.ap-northeast-1.amazonaws.com https://routes.geo.ca-central-1.amazonaws.com https://routes.geo.eu-central-1.amazonaws.com https://routes.geo.eu-west-1.amazonaws.com https://routes.geo.eu-west-2.amazonaws.com https://routes.geo.eu-north-1.amazonaws.com https://routes.geo.sa-east-1.amazonaws.com https://geofencing.geo.us-east-1.amazonaws.com https://geofencing.geo.us-east-2.amazonaws.com https://geofencing.geo.us-west-2.amazonaws.com https://geofencing.geo.ap-southeast-1.amazonaws.com https://geofencing.geo.ap-southeast-2.amazonaws.com https://geofencing.geo.ap-northeast-1.amazonaws.com https://geofencing.geo.ca-central-1.amazonaws.com https://geofencing.geo.eu-central-1.amazonaws.com https://geofencing.geo.eu-west-1.amazonaws.com https://geofencing.geo.eu-west-2.amazonaws.com https://geofencing.geo.eu-north-1.amazonaws.com https://geofencing.geo.sa-east-1.amazonaws.com https://*.auth.us-east-1.amazoncognito.com https://*.auth.us-east-2.amazoncognito.com https://*.auth.us-west-2.amazoncognito.com https://*.auth.ap-southeast-1.amazoncognito.com https://*.auth.ap-southeast-2.amazoncognito.com https://*.auth.ap-northeast-1.amazoncognito.com https://*.auth.ca-central-1.amazoncognito.com https://*.auth.eu-central-1.amazoncognito.com https://*.auth.eu-west-1.amazoncognito.com https://*.auth.eu-west-2.amazoncognito.com https://*.auth.eu-north-1.amazoncognito.com https://*.auth.sa-east-1.amazoncognito.com https://cognito-idp.us-east-1.amazonaws.com https://cognito-idp.us-east-2.amazonaws.com https://cognito-idp.us-west-2.amazonaws.com https://cognito-idp.ap-southeast-1.amazonaws.com https://cognito-idp.ap-southeast-2.amazonaws.com https://cognito-idp.ap-northeast-1.amazonaws.com https://cognito-idp.ca-central-1.amazonaws.com https://cognito-idp.eu-central-1.amazonaws.com https://cognito-idp.eu-west-1.amazonaws.com https://cognito-idp.eu-west-2.amazonaws.com https://cognito-idp.eu-north-1.amazonaws.com https://cognito-idp.sa-east-1.amazonaws.com https://h57jdz559a.execute-api.us-west-2.amazonaws.com"
```

Comment:
Can we use NL_BASE_URL here as well?

Reply:
Ive tried to use the env variables to enter the url but it doesn't seem to work. Ive also tried some solutions online but no luck.